WatchGuard patches to sources for curl-7.58.0 as of
Mon May  9 13:50:04 PDT 2022
The patches shown here have been applied to source .tar.gz 
files supplied with the WatchGuard Open Source Archive.

==========================================================================
--- curl-7.58.0/ltmain.sh.orig	2022-05-09 13:49:49.969437826 -0700
+++ curl-7.58.0/ltmain.sh	2022-05-09 13:49:51.373380847 -0700
@@ -8367,7 +8367,8 @@
 	      fi
 	    else
 	      # We cannot seem to hardcode it, guess we'll fake it.
-	      add_dir=-L$libdir
+	      # WG: don't fake it this introduces host pollution
+	      # add_dir=-L$libdir
 	      # Try looking first in the location we're being installed to.
 	      if test -n "$inst_prefix_dir"; then
 		case $libdir in
